topics/single-cell/tutorials/scrna-case_FilterPlotandExplore_SeuratTools/tutorial.md

```diff
@@ -64,7 +64,7 @@ We’ll provided you with experimental data to analyse from a mouse dataset of f
 ## Get Data onto Galaxy
 To start, let's get our dataset loaded into Galaxy.
 
-{% include _includes/cyoa-choices.html option1='EBI Data Retrieval' option2='Importing from a history' option3='Uploading from Zenodo' default='EBI-Data-Retrieval' text="There are multiple ways in which to collect the data for this tutorial. I find it easiest to do so via the EBI Data Retrieval." %}
+{% include _includes/cyoa-choices.html option1='EBI-Data-Retrieval' option2='Importing-from-a-history' option3='Uploading-from-Zenodo' default='EBI-Data-Retrieval' text="There are multiple ways in which to collect the data for this tutorial. I find it easiest to do so via the EBI Data Retrieval." %}
```

Comment:
this will be user visible then, not sure if that's desired. (In the process from option name to class, spaces are replaced with minus, so, you may want to retain the spaced version for the option text labels)

Reply:
:+1: then I will revert it